core/domain/feedback_domain_test.py

```diff
@@ -119,6 +118,19 @@ def test_to_dict(self) -> None:
             expected_message_dict, observed_message.to_dict())
 
 
+class TestFullyQualifiedMessageIdentifier(test_utils.GenericTestBase):
```

Comment:
Do we have to create new classes instead of just adding new functions to existing classes?

Reply:
I will add functions in existing classes.

Reply:
@gp201 i have created functions and removed new classes